crates/utils/serde-test-macros/src/lib.rs

```diff
@@ -22,25 +22,70 @@ use syn::{AttributeArgs, Ident, Item, Lit, Meta, MetaList, NestedMeta, Type, par
 /// * Implementation of `Arbitrary` trait
 /// * Implementations of `Serialize` and `DeserializeOwned` traits
 ///
-/// For testing generic types, use the `types(...)` attribute to list type parameters for testing,
-/// separated by commas. For complex types (e.g., ones where type parameters have their own
-/// parameters), enclose them in quotation marks. To test different combinations of type parameters,
-/// `types` can be used multiple times.
+/// # Configuration Attributes
 ///
-/// # Example
+/// The macro supports configuration attributes to control test generation:
+///
+/// | Attribute   | Type | Default | Purpose | Features Required |
+/// |-------------|------|---------|---------|-------------------|
+/// | `serde_test` | `bool` | `true` | Generate standard Serde round-trip tests | `arbitrary`, `serde`, `test` |
+/// | `winter_serde` | `bool` | `false` | Generate Winterfell-style round-trip tests | `arbitrary`, `test` |
```

Comment:
By "winterfell-style", we mean "using winterfell's `Serializable` and `Deserializable` traits, right? I would clarify that here, as this was not clear to me before reading the implementation